docs/exposing_dns.md

```diff
@@ -0,0 +1,71 @@
+# Exposing DNS UDP traffic for k8gb
+
+In order for k8gb to function properly, associated CoreDNS service deployed with k8gb needs be exposed for external DNS UDP traffic on cluster worker nodes.
+
+Actual ways to achieve this depend on many factors, such as underlying infrastructure (cloud, on-prem, managed vs bare-metal setup), means to expose CoreDNS service (ClusterIP, LoadBalancer),
+type of load balancer or ingress controller used etc.
+This topic is outside the project's scope, as often the related configuration is shared by cluster services, requires additional permissions, and as result can't be owned by k8gb controller deployment.
+However, we can describe a few examples using common Kubernetes configurations, which have been thoroughly tested in local and production environments.
+
+## Ingress Controller with UDP support (NGINX)
+
+> *Check [NGINX Ingress controller official documentation](https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/) for additional information*
+
+In general, an Ingress resource doesn't support TCP or UDP services. In order to let NGINX Ingress controller know that we want to expose UDP port for k8gb CoreDNS service (`k8gb-coredns`), we need to create or patch `udp-services` ConfigMap in a  namespace where NGINX ingress controller is installed (`ingress-nginx` by default).
+This can be done by applying the following manifest:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: udp-services
+  namespace: ingress-nginx
+data:
+  53: "k8gb/k8gb-coredns:53"
```

Comment:
Is it worth to mention namespace and chart release name?

Reply:
@k0da , amended with instructions on how to identify the associates service, please check.